include/fplus/side_effects.hpp

```diff
@@ -269,6 +269,20 @@ auto execute_max_n_times_until_success(std::size_t n,
         replicate(n, effect_to_std_function(eff)));
 }
 
+// API search type: execute_n_times : (Int, Io a)) -> Io Void
```

Comment:
I think the other functions use `Io ()` instead of `Io Void`. :)

Reply:
ok, I changed this